llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2642,6 +2694,11 @@ struct VPWidenStoreEVLRecipe final : public VPWidenMemoryRecipe {
     setMask(Mask);
   }
 
+  VPWidenStoreEVLRecipe *clone() override {
+    llvm_unreachable("VPWidenStoreEVLRecipe cannot be cloned");
```

Comment:
I don't understand the purpose of this interface. I want to know why it is set to unreachable

Reply:
[Florian's comment on this function](https://github.com/llvm/llvm-project/pull/93854#discussion_r1643384493). If you do have a use of a clone function for other cases, it will be great to have it in upstream.